README.md

```diff
@@ -5,7 +5,10 @@ Reservation component on the Open Table restaurant page.
 ### Initial Database Seeding
 
 1. Run the following script in terminal with your MySQL password within this project's root directory. This script will create the `reservation` database in your mySQL server. 
-    `mysql < ./server/db/Schema.sql -u root -p`
+    `mysql --max_allowed_packet=100M -u root -p < ./server/db/Schema.sql`
```

Comment:
@LittleJennie 
@Sam-I-Am92 

Parteek--maybe you didn't have any questions about why someone would need to do this---but I do! Jennie, could you share what type of blob, string, or command you are sending is so large that the packet size needs to be increased?

Reply:
I have seeding about 56k of data to database. We have 100 restaurants to render, and I am allocating 6 time slots for the next 90 days for each restaurant.